packages/wasm/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
MIT please: https://www.notion.so/sentry/Licensing-b99b3c083e174ed0850c65e313a79a68

Reply:
I know but everything else in the repo appears the same right now.

Reply:
New packages like Vue are already MIT, please also change this.